ReleaseProcess.md

```diff
@@ -0,0 +1,28 @@
+# The Windows Device Portal Wrapper Release Process
+
+The following description and diagram illustrate the process that the Windows Device Portal Wrapper repository undergoes to create a release.
+
+![Release Process Flow](External/ReadMeImages/ReleaseProcess.png)
+
+## Release Steps
+1. Declare that a release is feature complete.
+
+This will done by / in coordination with the Windows Device Portal Wrapper maintainers.
+
+2. Create a stabilization branch (ex: Stabilization_v1.2.3) off of Development
+
+Bug fixes get submitted to the stabilization branch while work on the next release continues in Development. Note: While not expected in every release, there _may_ be some cherry-picking of individual commits from Development.
+
+3. When the release has stabilized, a pull request is issued from the stabilization branch to master and it is merged.
```

Comment:
Is there any guidance on how long the stabilization period is? For MixedRealityToolkit-Unity, we say it's 1 week.